cumulus_library/template_sql/core_medication.sql.jinja

```diff
@@ -23,7 +23,11 @@ CREATE TABLE core__medication AS (
             t.r.code,
             t.r.display,
             t.r.system AS code_system,
+            {% if missing_userselected %}
+            false AS userselected
+            {% else %}
             t.r.userselected
+            {% endif %}
```

Comment:
i think? this case is only present in the synthea dataset, but it doesn't hurt to check for (and, y'know, allows us to test against synthea data).